python/python-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     tier: frontend
 spec:
   replicas: 1
+  selector:
```

Comment:
Will it trigger installing newest skaffold for all the folks?
Do we need to do it manually every time there is a new version of skaffold?
What happens if we don't do it and folks still update to newest skaffold? Is it all backward compatible?